web/src/routes/(user)/people/+page.svelte

```diff
@@ -19,6 +19,7 @@
   import ImageThumbnail from '$lib/components/assets/thumbnail/image-thumbnail.svelte';
   import { onDestroy, onMount } from 'svelte';
   import { browser } from '$app/environment';
+  import SuggestMerge from '$lib/components/faces-page/merge-suggestion-modal.svelte';
```

Comment:
Can we change the name of the component to match the file name as well? `MergeSuggestionModal`